apps/artemis-web/components/footer/footer.tsx

```diff
@@ -22,7 +23,7 @@ class FooterComponent extends React.Component<MyProps> {
           style={{ marginLeft: '0px', paddingLeft: '8.333333%' }}
           maxWidth="lg"
         >
-          ARTEMIS v.'latest@'
+          ARTEMIS v.'{process.env.NEXT_PUBLIC_SYSTEM_VERSION}@{process.env.NEXT_PUBLIC_REVISION || 'HEAD'}'
```

Comment:
Use `SYSTEM_VERSION` from `.env`. Also, where do you get the `NEXT_PUBLIC_REVISION`? Could you instead get the latest github revision from sth like `git rev-parse HEAD`? So the string will look like `ARTEMIS v.'SYSTEM_VERSION@latest_commit_id`, which gives us all the info we actually need.

Reply:
I modified the entrypoint scripts so that the revision is added in the environmental vars dynamically.